org.lflang/src/org/lflang/TargetConfig.java

```diff
@@ -129,159 +131,161 @@
+    public List<String> fileNames = new ArrayList<>();
 
     /**
      * List of file names from the files target property with no path info.
      * Useful for copying them to remote machines. This is needed because
      * target files can be resources with resource paths.
      */
-    public List<String> filesNamesWithoutPath = newLinkedList;
+    public List<String> filesNamesWithoutPath =new ArrayList<>();
 
     /**
      * If true, configure the execution environment to keep executing if there
      * are no more events on the event queue. The default is false.
      */
-    public boolean keepalive = false
+    public boolean keepalive = false;
 
     /**
      * The level of logging during execution. The default is INFO.
      */
-    public LogLevel logLevel = LogLevel.INFO
+    public LogLevel logLevel = LogLevel.INFO;
 
     /**
      * Flags to pass to the linker, unless a build command has been specified.
      */
-    public String linkerFlags = ""
+    public String linkerFlags = "";
 
     /**
      * If true, do not invoke the target compiler or build command.
      * The default is false.
      */
-    public boolean noCompile = false
+    public boolean noCompile = false;
 
     /**
      * If true, do not perform runtime validation. The default is false.
      */
-    public boolean noRuntimeValidation = false
+    public boolean noRuntimeValidation = false;
 
     /**
      * List of proto files to be processed by the code generator.
      */
-    public List<String> protoFiles = newLinkedList
+    public List<String> protoFiles = new ArrayList<>();
 
     /**
      * The version of the runtime library to be used in the generated target. 
      */
-    public String runtimeVersion = null
+    public String runtimeVersion = null;
 
     /**
      * The number of worker threads to deploy. The default is zero (i.e.,
      * all work is done in the main thread).
      */
-    public int threads = 0
+    public int threads = 0;
 
     /**
      * The timeout to be observed during execution of the program.
      */
-    public TimeValue timeout
+    public TimeValue timeout;
 
     /**
      * If non-null, configure the runtime environment to perform tracing.
      * The default is null.
      */
-    public TracingOptions tracing = null
+    public TracingOptions tracing = null;
 
-}
 
-/**
- * Settings related to clock synchronization.
- */
-class ClockSyncOptions {
-    
-    /**
-     * FIXME
-     * The default is 10.
-     */
-    public int attenuation = 10
 
     /**
-     * Whether or not to collect statistics while performing clock synchronization.
-     * This setting is only considered when clock synchronization has been activated.
-     * The default is true.
+     * Settings related to clock synchronization.
      */
-    public boolean collectStats = true
+    public static class ClockSyncOptions {
 
-    /**
-     * FIXME
-     */
-    public boolean localFederatesOn
+        /**
+         * FIXME
```

Comment:
@Soroosh129: maybe you could quickly jump in and address these FIXMEs?

Reply:
What is this FIXME referring to?

Reply:
Missing comments.